### DISCUSSION TITLE
Service Level Agreement

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#79] [Added]
### What are some documents containing these terms?

| Example | Service Name | URL | Title | Jurisdiction(s) |
|:----|:----|:----|:------|:-------------|
| 1 | BIT | [URL 🔗](https://www.bit.nl/en/about-bit/terms-and-policies/sla-addendum-colocation-bit-2?from=nl) | Service level agreement | 🌐 |
| 2 | Own | [URL 🔗](https://www.owndata.com/legal/sla) | Service Level addendum | 🌐 |
| 3 | Vultr | [URL 🔗](https://www.vultr.com/legal/sla/) | Service Level Agreement | 🌐 |
| 4 | Wasabi | [URL 🔗](https://wasabi.com/legal/sla) | Service Level Agreement | 🌐 |
| 5 | Asana | [URL 🔗](https://asana.com/service-levels) | Service Level Addendum | 🌐 |

### What could be some alternative names for this type?

Service Level Addendum

### Whose rights and duties are defined in these terms?

end user

### What is the object of commitment in these terms?

define and guarantee specific levels of service performance, availability, and quality

### Are there any relevant references for this type?

[Wikipedia](https://en.wikipedia.org/wiki/Service-level_agreement)

**Comment by Ndpnt** [comment]
One example is missing to move this discussion as `Qualified`

**Reply by clementbiron** [reply]
I've added one 👍
